lib/util/git/types.ts

```diff
@@ -9,7 +9,8 @@ export interface GitAuthor {
 
 export type GitNoVerifyOption = 'commit' | 'push';
 
-export type CommitSha = string;
+// We want to make sure it's a long sha of 40 characters and not just a string
```

Comment:
use tsdoc comment style

Reply:
☝️

Reply:
Finally got time to update :) It's now a tsdoc comment style